src/runtime_src/core/common/utils.cpp

```diff
@@ -49,6 +49,14 @@ precision(double value, int p)
 
 namespace xrt_core { namespace utils {
 
+std::string
+get_hostname()
```

Comment:
get_hostname is now in here! It parses the OS ptree and returns the hostname. the message and SubCmdConfigure files call this function to get the data.